core/src/main/scala/chisel3/experimental/package.scala

```diff
@@ -58,7 +58,7 @@ package object experimental {
   /** Requires that a node is hardware ("bound")
     */
   object requireIsHardware {
-    def apply(node: Data, msg: String = ""): Unit = {
+    def apply(node: IsBindable, msg: String = ""): Unit = {
```

Comment:
so for a property, is "hardware" the same definition (it must be a value e.g. bound to a Port, not a chisel type?) perhaps we should change this name or put a todo to be "value" not "hardware"

Reply:
Yeah, the name "hardware" is not really applicable to non-Data types, but the meaning of requireIsHardware and requireIsChiselType are the same for Data and non-Data.